site/styles/globals.scss

```diff
@@ -3,3 +3,13 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.app-width-container {
```

Comment:
These styles should be applied using tailwind classes

Reply:
done